### ISSUE TITLE
Syntax error.

### ISSUE THREAD

**Issue by kentonself-appdynamics** [Rookout/tutorial-python#45]
Hi!

I'm trying to get this working and hitting this:

`$ python app.py

Traceback (most recent call last):

  File "app.py", line 13, in <module>

    from utils.logging import on_add_todo_logging, on_get_todos_logging

  File "/home/ubuntu/tutorial-python/utils/logging.py", line 13

    logger.debug(f"The user needs to do - @@@@@@@@ {todoStr} @@@@@@@@")

                                                                     ^
SyntaxError: invalid syntax`

The "f" between the "(" and the open parenthesis looks suspicious, but I am not very experienced with Python.

**Comment by ElDuderinos** [comment]
Hi @kentonself-appdynamics, so this isn't actually a syntax error. That's us using f-string (https://realpython.com/python-f-strings/) which is relatively new for Python, I assume that you didn't use Python 3.6+.
Anyway, thanks to your input I've removed the usage of f-strings and used the good ole' classic Python string formatting.
See https://github.com/Rookout/tutorial-python/pull/46. Things should now work flawlessly for you.
Let us know if you have any other issues, I'll be closing this one.

**Comment by kentonself-appdynamics** [comment]
Oh, Thank you!

And when you said it was python 3, I realized that perhaps I was running python 2. The README file mentions running python app.py, and I did not think to make that call python3.

--

Kenton Self| Sr. Sales Engineer
Office: +1 469-255-2671
Mobile: +1 469-831-1696
https://www.linkedin.com/in/kentonself/
Email: kenton.self@appdynamics.com<mailto:kenton.self@appdynamics.com>

AppDynamics
The Application Intelligence Company
